src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -816,12 +816,12 @@ void RegisterAppInterfaceRequest::SendRegisterAppInterfaceResponseToMobile() {
                                           &(msg_params[strings::app_hmi_type]));
   }
 #endif  // SDL_REMOTE_CONTROL
+  SendResponse(true, result_code, add_info.c_str(), &response_params);
```

Comment:
@tkireva , did you make these changes with `REMOTE_CONTROL OFF` ? if yes, so these fix will affect Reverse SDL